src/guides/v2.3/install-gde/bk-install-guide.md

```diff
@@ -5,6 +5,8 @@ functional_areas:
   - Install
   - System
   - Setup
+redirect_from:
+  - /guides/v2.4/install-gde/continue-to-install.html 
```

Comment:
I don't understand this redirect. I don't usually see a 2.4-> 2.3 version change in these.

Reply:
Good catch!